phpBB/adm/style/acp_search_index.html

```diff
@@ -0,0 +1,80 @@
+{% include 'overall_header.html' %}
+
+<a id="maincontent"></a>
+
+<script>
+// <![CDATA[
+	/**
+	* Popup search progress bar
+	*/
+	function popup_progress_bar(progress_type)
+	{
+		close_waitscreen = 0;
+		// no scrollbars
+		popup('{{ UA_PROGRESS_BAR }}&amp;type=' + progress_type, 400, 240, '_index');
+	}
+// ]]>
+</script>
+
+<h1>{{ lang('ACP_SEARCH_INDEX') }}</h1>
+
+<p>{{ lang('ACP_SEARCH_INDEX_EXPLAIN') }}</p>
+
+{% for backend in loops.backend %}
+
+	{% if backend.S_STATS is not empty %}
+
+	<form id="acp_search_index_{{ backend.TYPE }}" method="post" action="{{ U_ACTION }}">
```

Comment:
No, I fixed it, it was already type before, I just corrected the name.
The variable name was referencing the type
https://github.com/phpbb/phpbb/blob/master/phpBB/includes/acp/acp_search.php#L378

But I don't know what is better, because name have spaces (and id's must not contain spaces), and the type contains `\` character

Reply:
Ok, then I will resolve this

Reply:
Yes, seems valid in html5
https://stackoverflow.com/questions/70579/what-are-valid-values-for-the-id-attribute-in-html